Cartfile.private

```diff
@@ -1,2 +1,2 @@
-github "thoughtbot/Curry"
+github "thoughtbot/Curry" "tvos"
 github "thoughtbot/Runes" >= 3.0.0
```

Comment:
@nickygerritsen  you can now point to this branch of Runes below, as https://github.com/thoughtbot/Runes/pull/51 has been merged and there's a new tvos branch that shares the scheme:

github "thoughtbot/Runes" "tvos"